test/runtests.jl

```diff
@@ -23,4 +23,8 @@ end
 @test tst1[1:2] == tst1[0x01:0x02] == tst1[[1,2]] == "an"
 
 # issue #23 — will change if #17 is addressed
-@test L"x" * L"y" == "\$x\$\$y\$"
\ No newline at end of file
+@test L"x" * L"y" == "\$x\$\$y\$"
+
+# show should return nothing
+@test show(IOBuffer(), MIME"application/x-latex"(), tst1) === nothing
```

Comment:
(Note that you can just pass the string `"application/x-latex"` for the second argument; no need to construct a `MIME` object.)